code/__HELPERS/_string_lists.dm

```diff
@@ -1,3 +1,4 @@
+#define STRING_DIRECTORY "strings"
```

Comment:
`#undef` at the end

Reply:
I'll move it to the global define folder